server/chat-plugins/helptickets.js

```diff
@@ -69,6 +70,25 @@ function writeTickets() {
 	));
 }
 
+/**
+ * @param {string} line
+ */
+function writeStats(line) {
+	// ticketType,totalTime,timeToFirstClaim,inactiveTime,resolution,result,staff|userids|seperated|with|bars
+	const date = new Date();
+	const month = Chat.toTimestamp(date).split(' ')[0].split('-', 2).join('-');
+	let ticketStats = '';
+	try {
+		ticketStats = FS(`logs/tickets/${month}.csv`).readSync();
+	} catch (e) {
+		if (e.code !== 'ENOENT') throw e;
+	}
+	ticketStats += (ticketStats ? '\n' : '') + line;
```

Comment:
Writing the entire file contents to it every time sounds a bit inefficient (unless if I'm misunderstanding the code here), why not use something like `FS.append` since we're only adding one line at at time?